cmd/crd.go

```diff
@@ -0,0 +1,32 @@
+package cmd
+
+import (
+	"github.com/spf13/cobra"
+
+	"github.com/dominodatalab/forge/internal/crd"
+)
+
+var (
+	applyCrdCmd = &cobra.Command{
+		Use:   "crd-apply",
+		Short: "Apply the ContainerImageBuild CRD to a cluster",
+		Run: func(cmd *cobra.Command, args []string) {
+			if err := crd.Apply(); err != nil {
+				panic(err)
```

Comment:
Do we want to `panic` here, return the err, or handle it differently? Same question for `crd-delete` command.

Reply:
Yes, I should use RunE, thanks!